infra/src/Pos/Infra/Reporting/Wlog.hs

```diff
@@ -65,8 +64,11 @@ readWlogFile logConfig = case mLogFile of
   where
     -- Grab all public log files, using the 'LoggerConfig', and take the
     -- first one.
-    allFiles = map snd $ retrieveLogFiles logConfig
-    mLogFile = (fmap fst . uncons) (filter (".pub" `isSuffixOf`) allFiles)
+    basepath = fromMaybe "./" $ logConfig ^. lcBasePath
+    allFiles = map ((</> basepath) . snd) $ retrieveLogFiles logConfig
+    mLogFile = case filter (".pub" `isSuffixOf`) allFiles of
```

Comment:
this will not work anymore as the file names contain the timestamp of file creation.
=> add 'pub' subdirectory to basepath

the collecting, zipping and sending of the log files is now being done in Daedalus.
=> opened CBR-421 which will remove this code